backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/nodes/UsageNodeDescriptionProvider.java

```diff
@@ -54,7 +54,10 @@ protected String getSemanticCandidatesExpression(String domainType) {
 
     @Override
     protected String createPreconditionExpression() {
-        // Actors are handled with a different NodeDescription, see ActorNodeDescriptionProvider.
+        // Actors are handled with a different NodeDescription: ActorNodeDescriptionProvider.
+        // We can't represent actors with conditional styles because they require to remove inside labels and add
+        // outside labels, and set the keepAspectRatio property, which are all defined in the NodeDescription and not in
+        // the style.
```

Comment:
Improved the comment here to make clear why we can't represent an Actor using the `UsageNodeDescriptionProvider`